### PR TITLE
less terse about status code

### LINKED ISSUE #85: clarify http errors vs dns errors
largely a restating of mailing list thread: A question on the mix of DNS and HTTP semantics

>  I'm certainly happy to add a sentence or two to help make that more clear (suggestions?) but I don't want to start defining/reiterating HTTP semantics in the DoH doc - we've already got lots of documents (i.e. rfc723x.. which are all included normatively by 7230 which doh includes normatively) that do that and its not in doh's charter to mess those things anyhow.

### PR DESCRIPTION
**Pull request by mcmanus**
intended closes #85

### CODE REVIEW

File: draft-ietf-doh-dns-over-https-latest.mkd [hunk 1]
```diff
@@ -277,7 +277,13 @@ accept = application/dns-udpwireformat
 An HTTP response with a 2xx status code ({{RFC7231}} Section 6.3)
 indicates a valid DNS response to the query made in the HTTP
 request. A valid DNS response includes both success and failure
-responses.
+responses. For example, a DNS failure response such as SERVFAIL or
+NXDOMAIN will contain a successful HTTP 2xx status code even though
```
Comment (martinthomson) [inline draft-ietf-doh-dns-over-https-latest.mkd]: Nit: The NXDOMAIN/SERVFAIL won't contain the status code.

File: draft-ietf-doh-dns-over-https-latest.mkd [hunk 2]
```diff
@@ -277,7 +277,13 @@ accept = application/dns-udpwireformat
...
+there was a failure at the DNS layer. Responses with non successful
```
Comment (martinthomson) [inline draft-ietf-doh-dns-over-https-latest.mkd]: hypen in non-successful

File: draft-ietf-doh-dns-over-https-latest.mkd [hunk 3]
```diff
@@ -277,7 +277,13 @@ accept = application/dns-udpwireformat
...
+HTTP status codes do not contain DNS answers to the question in the
+corresponding request. Some of these non-successful HTTPS responses
+(e.g. redirects or authentication failures) may allow clients to make
```
Comment (martinthomson) [inline draft-ietf-doh-dns-over-https-latest.mkd]: s/may/could/

File: draft-ietf-doh-dns-over-https-latest.mkd [hunk 4]
```diff
@@ -277,7 +277,13 @@ accept = application/dns-udpwireformat
...
+new requests to satisfty the original question.
```
Comment (martinthomson) [inline draft-ietf-doh-dns-over-https-latest.mkd]: typo: "satisfy"

File: draft-ietf-doh-dns-over-https-latest.mkd (hunk 1 above)
Comment (vttale) [reply draft-ietf-doh-dns-over-https-latest.mkd]: How dreadfully embarrassing for me.  LPT: don't review PRs during meetings

**Pull request merged**